### ISSUE TITLE
[Dropdown] onChange method getting called during initial render of component

### ISSUE THREAD

**Issue by kernelbat** [kyndryl-design-system/shidoka-applications#99]
**Describe the bug**

onChange method getting called during initial render of component

**To Reproduce**
Component I have used:

  <Dropdown
          name='account'
          required = {true}
          placeholder={"Placeholder"}
          value={value}
          onChange={ setFormData}
        >
          <span slot='label'>{'Account Type'}</span>
          <DropdownOption value={'customer'}>Customer</DropdownOption>
          <DropdownOption value={'provider'}>Provider</DropdownOption>
  </Dropdown>

**Expected behavior**

onchange method should not be called when component gets mounted.

**Comment by brian-patrick-3** [comment]
Hi there. This happens because the dropdown value is derived initially from aggregating the child dropdown-options which have `selected` attributes. Is this causing some sort of problem for you in practice?

**Comment by kernelbat** [comment]
> Hi there. This happens intentionally because the dropdown initial value is derived initially from aggregating the child dropdown-options which have `selected` attributes. Is this causing some sort of problem for you in practice?

Yes, it mainly causes error when we have default value and based on that other components are getting rendered. The value gets reset on initial render

**Comment by brian-patrick-3** [comment]
> Yes, it mainly causes error when we have default value and based on that other components are getting rendered. The value gets reset on initial render

You have to set the initial/default value by setting `selected` on the child options.
